wiki/Scoping-a-model.md

```diff
@@ -0,0 +1,70 @@
+
+### General points
+
+(some inspired by <https://dataingovernment.blog.gov.uk/2017/09/04/introducing-agile-into-an-established-data-science-team/> )
+
+#### what is the problem we're trying to solve?
+This is not trivial: it takes some interviewing people to get to a pain they experience on a regular basis.  It pays off to use drilling down techniques like the 5 whys (<https://en.wikipedia.org/wiki/5_Whys>) - sometimes the pain people experience has a non-obvious cause.
+
+Another angle of approach is to look at it from a quantitative perspective: what will make them more effective, what will save more lives, what will save more money.  This also takes some understanding of the processes involved: the initial interviews should be about understanding the process in general.  What are the costs?
+
+Thirdly we have our own objective: we need a model:
+
+* they couldn't make by themselves in excel: that leverages our expertise
+* that we can use for other customers: generic enough
+* that we can demonstrate makes a difference, from decision-maker level (to be able to sell it).
+
+
+#### who are our users?
+
+This is interlinked with the problem question of previous paragraph.  We need to find fairly quickly who will use our model.  They may not be the same people who we talk to initially (decision-maker level) but they _will_ be the people we need to talk to to get the necessary domain knowledge.  Relevant heads of services might be the right people to talk to.
+
+#### what constraints are we operating within?
+
+There are sometimes legal constraints, it pays to find out which they are fairly early on (like for instance privacy).
+
+#### what is the technology legacy and who else has solved this problem
+
+How did they do it so far?
+Do we have competitors?
+
+#### what data sources do we have access to?
+
+As mentioned in the agile data science manifesto (<https://www.oreilly.com/ideas/a-manifesto-for-agile-data-science>) - this is also one of the 'stakeholders' we need to listen to.  Which data we have access to and what it reveals also imposes its own constraints on what we can do.  A minimum of exploratory data analysis is always part of the scope process.
+
+It is important not to spend to much time on this, but to focus on the scope determination.
+
+A major caveat here is that getting data takes time.  To quote the [blog post](https://dataingovernment.blog.gov.uk/2017/09/04/introducing-agile-into-an-established-data-science-team/)
+
+>Some organisations don’t even know what data they hold. Others have lots of data but it’s in such bad shape that it’s very hard to work with. Still more have good data but it’s impossible to get access to it for security reasons. Understanding all of these in advance of starting a project greatly increases the chances of success.
+
+While waiting it might be a good idea to have other projects lined up so as not to waste time before we have a scope.
```

Comment:
Yes! Don't commit time until the data is in hand! We've used up budget on this before